lib/cli.rb

```diff
@@ -322,6 +322,19 @@ def list_member_boards
     }
   end
 
+  desc "after-planning", "Move remaining cards to backlog"
+  long_desc <<EOT
+EOT
```

Comment:
There is a description missing. There either should be one or we don't need the `long_desc` tag, do we?